src/main/java/nl/knaw/dans/bridge/plugin/lib/common/ITransform.java

```diff
@@ -0,0 +1,17 @@
+package nl.knaw.dans.bridge.plugin.lib.common;
+
+import nl.knaw.dans.bridge.plugin.lib.exception.BridgeException;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+/*
+ * @author: Eko Indarto
+ */
+public interface ITransform {
```

Comment:
A bit more documentation; it's extracting deposit information; the metadata for the destination archive and the list of (data) files that need to be deposited. Transformation (Xslt) is the means by which we accomplice that.